Fridge tutorial

ai-fast-track opened the pull request:
I added an new notebook called `02-how_train_dataset` that shows how to train a dataset. The example used is the one of the Fridge Objects Dataset. I used the new API. As a consequence, I had to update the 01_getting_started NB as well as the README file.

lgvaz commented on mantisshrimp/datasets/fridge/transforms.py:
```diff
@@ -0,0 +1,61 @@
+    return AlbuTransform(
+        [
+            A.LongestMaxSize(384),
+            A.RandomSizedBBoxSafeCrop(320, 320, p=0.3),
```
I believe LongestMaxSize will not work, because efficient-det needs a fixed squared input that also needs to be divisible by 128

ai-fast-track commented:
I made these changes:
- deleted `from .rcnn_learner import *` from `fastai.learner.__init__`: because `rcnn_learner` file has been deleted
- updated the dataloaders calls:

Tested on Colab: It works!
```
from mantisshrimp.models.rcnn.faster_rcnn import dataloaders

# Create both training and validation dataloaders
train_dl = dataloaders.train_dataloader(train_ds, batch_size=16, num_workers=4, shuffle=True)
valid_dl = dataloaders.valid_dataloader(valid_ds, batch_size=16, num_workers=4, shuffle=False)
```

lgvaz commented:
What is left on this one? The transforms?

ai-fast-track commented:
Yes the tranforms and updating the predictions section by using the new Prediction API.

ai-fast-track commented:
Outdated API
